docs/rules/prefer-const.md

```diff
@@ -0,0 +1,50 @@
+# Suggest to use `const` (prefer-const)
+
+If a variable is never changed, using the `const` declaration is better.
+
+## Rule Details
+
+The following patterns are considered warnings:
+
+```js
+let a = 3;
+console.log(a);
+```
+
+```js
+for (let a of [1,2,3]) { // `a` is re-defined (not changed) on each loop step.
+    console.log(a);
+}
+```
+
+The following patterns are not considered warnings:
+
+```js
+let a; // there is not the initializer.
+console.log(a);
+```
+
+```js
+for (let i = 0, end = 10; i < end; ++i) { // `end` is never changed, but we cannot separate the declaration without changing scope.
+    console.log(a);
+}
+```
+
+```js
+for (let a in [1,2,3]) { // `a` is changed by each loop step.
+    console.log(a);
+}
+```
+
+```js
+var a = 3; // suggest to use `no-var` rule.
+console.log(a);
+```
+
+## When Not to Use It
+
+If you don't want to be notified about never change variables, you can safely disable this rule.
```

Comment:
"if you don't want to be notified about variables that are never modified after initial assignment"